src/test/scala/org/hammerlab/guacamole/util/TestUtil.scala

```diff
@@ -27,7 +27,7 @@ import org.apache.spark.SparkContext
 import org.hammerlab.guacamole.ReadSet
 import org.hammerlab.guacamole.loci.set.LociParser
 import org.hammerlab.guacamole.pileup.Pileup
-import org.hammerlab.guacamole.reads.{InputFilters, MappedRead, MateAlignmentProperties, PairedRead, Read, ReadLoadingConfig}
+import org.hammerlab.guacamole.reads._
```

Comment:
nit: I prefer avoiding wildcard imports except when importing `implicit def`s from `object`s. You can prevent IntelliJ from using wildcard imports here:

![](http://cl.ly/012k2M0c3P35/Screen%20Shot%202016-05-09%20at%205.32.04%20PM.png)

will add to #439 if you agree